alf/examples/sac_carla_conf.py

```diff
@@ -60,17 +63,50 @@
 
 alf.config(
     'CarlaEnvironment',
-    vehicle_filter='vehicle.audi.tt',
+    vehicle_filter='vehicle.*',
     num_other_vehicles=20,
     num_walkers=20,
     # 1000 second day length means 4.5 days in replay buffer of 90000 length
     day_length=1000,
     max_weather_length=500,
 )
 
-alf.define_config('taac', False)
+
+class NormalizedActionWrapper(AlfEnvironmentBaseWrapper):
+    """An AlfEnvironment wrapper that converts the last action dim ('reverse') from
+    [0,1] to [-1,1].
```

Comment:
This is because TaacL and TaacQ both require action to be in [-1,1]

Reply:
This requirement needs to be checked in TaacAlgorithm

Reply:
> This requirement needs to be checked in TaacAlgorithm



> This requirement needs to be checked in TaacAlgorithm

TaacAlgorithm is fine with any action spec. Only TaacL and TaacQ needs this requirement and they're already checked.

Reply:
> > This requirement needs to be checked in TaacAlgorithm
> 
> TaacAlgorithm is fine with any action spec. Only TaacL and TaacQ needs this requirement and they already assert this.